setup.py

```diff
@@ -47,6 +47,7 @@
     'Flask-CeleryExt>=0.1.0',
     'Flask-CLI>=0.2.1',
     'Flask-Mail>=0.9.1',
+    'fake-factory>=0.5.3',
```

Comment:
Why would you want to make writing tests more cumbersome?
Unless we're testing the validation part of user registration/creation, the actual values used are irrelevant.